src/chapters/classes-and-objects/inheritance.rst

```diff
@@ -0,0 +1,16 @@
+Inheritance and Other Topics
+============================
+
+.. todo:: We need to have a whole Ed Team discussion about whether to include these topics at all.
```

Comment:
The jury is still out on this.

For now, I'll remove the TODO, but I'd love to hear your take on whether or not to include inheritance.

If we vote to include it, I'll open a new branch to add content to the appendix.  Otherwise, the branch will remove this page and the appendix.

Reply:
I don't have an opinion one way or another but it looks like there are still some todos in the inheritance pages anyways

Reply:
Yes, filling in the appendix is the next task after writing Assignment 4.